util.py

```diff
@@ -0,0 +1,80 @@
+# Enter your source directory between the quotes here
+SOURCE_DIRECTORY = ''
+
+assert not SOURCE_DIRECTORY == '', 'SOURCE_DIRECTORY not configured'
+
+print("Source directory configured as {}".format(SOURCE_DIRECTORY))
+
+import sys
+sys.path.insert(0, SOURCE_DIRECTORY + '/test/functional')
+
+#############################################################################
+
+import argparse
+import os
+
+from test_framework.test_framework import BitcoinTestFramework
+
+# TestWrapper utility class.
+class TestWrapper(BitcoinTestFramework):
+    """Wrapper Class for BitcoinTestFramework.
+
+    Provides the BitcoinTestFramework rpc & daemon process management
+    functionality to external python projects."""
```

Comment:
Might note here that the value of this class instead of just using `BitcoinTestFramework` is that weve provided a bunch of sane default settings as well as defaulting to setup 3 nodes. (at least that is my understanding of the raison d'etre for this class)

Reply:
Yes, BitcoinTestFramework sets defaults when reading command line arguments with argparse. Since we don't do that, we have to set the defaults in TestWrapper. Only these options are set can we run the setup() and shutdown().